### ISSUE TITLE
Compile a non-coffeescript version? 

### ISSUE THREAD

**Issue by Dascrilla** [yogiben/meteor-starter#46]
Any way thats possible?

**Comment by nicholasalanbrown** [comment]
+1 for this. I write in pure JS, but this is so awesome I'm almost tempted to learn CoffeeScript just to use it.

Any way to get a pure JS version?

**Comment by latitudehopper** [comment]
+1

**Comment by djbf** [comment]
exactly what @nicholasalanbrown said!  super nice work.

**Comment by yogiben** [comment]
This would almost certainly be a fork and not maintained by me. I prefer coffeescript in a big way.

I'd be happy to help someone who wants to take the lead.

**Comment by MauriSalazar** [comment]
I did a fork and used js2.coffee compiler to get vanilla javascript.

The project seems to be working fine, I'll need to review the syntax of those generated .js files.

https://github.com/MauriSalazar/meteor-starter

**Comment by fjaguero** [comment]
Yes, please. I always feel like [this](https://twitter.com/dam/status/599612587999776768) when this happen.

**Comment by srault95** [comment]
# Convert to js

```bash
  git clone https://github.com/yogiben/meteor-starter.git

  npm install -g coffee-script

  find meteor-starter-name "*.coffee" -exec coffee -b --no-header -c {} \;
```

**Comment by nicholasalanbrown** [comment]
@yogiben after looking deeper into the efficiencies of a JavaScript workflow, you've totally converted me. I'm using you're original repo now. Good thing I didn't get too far into this project :-)

**Comment by yogiben** [comment]
@nicholasalanbrown haha great to hear!
